src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java

```diff
@@ -734,16 +734,6 @@ public boolean checkCanDropLoot()
      */
     public abstract void setFleeingState(final boolean fleeing);
 
-    /**
-     * Setter for the citizen pose.
-     *
-     * @param pose the pose to set.
-     */
-    public void updatePose(final Pose pose)
```

Comment:
Pretty sure now if you relog or reload the chunk they are sleeping in they will stand in their beds

Reply:
? This is just a duplicate setter we were using lol :D

Reply:
Meant more in general the pr. Just commented on the first file. The pose methods come from a time the setPose method was protected. Did you test with reloading and similar stuff to avoid back crawling citizens, etc?

Reply:
logic did not change, it simply got moved to the proper place

Reply:
Yeah, but at the other place it was constantly checked without other AIs potentially blocking the check.